app/Http/Controllers/ProfileController.php

```diff
@@ -1,13 +1,47 @@
 <?php
 
 namespace App\Http\Controllers;
-
+use App\Models\User;
 use Illuminate\Http\Request;
 
 class ProfileController extends Controller
 {
-    public function show() // Tords, maybe put user id as argument. if walay argument, then profile sa user... just a suggestion
+    public function show($id) //get user's info using user id
+    {
+       $user = User::find($id);
+       return view('layouts.profile', [
```

Comment:
profile View should reside to its module.
view('users.profile') not on 'layouts.profile' since layout is for all.